languages/elixir/exercises/concept/exceptions/test/test_helper.exs

```diff
@@ -0,0 +1,2 @@
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true, seed: 0)
```

Comment:
Why the hardcoded seed?

Reply:
It runs the tests in the order I've written them. Just for prelim work. I'll be removing it before review